crowdsec-docs/unversioned/getting_started/installation/linux.mdx

```diff
@@ -0,0 +1,146 @@
+---
+id: linux
+title: Linux
+pagination_prev: getting_started/pre_requisites
+pagination_next: getting_started/next_steps
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import CodeBlock from '@theme/CodeBlock';
+
+# Linux
+
+Before getting started it is advised to read the [prerequisites](/unversioned/getting_started/prerequisites.md) page to understand the requirements for running CrowdSec.
```

Comment:
We should as well refer the user to a general presentation of the project here (cf. `[1]` in the introduction page)